frame/election-provider-multi-phase/test-election-provider/Cargo.toml

```
@@ -0,0 +1,40 @@
+[package]
+name = "pallet-election-provider-test"
```

nit: 

```suggestion
name = "pallet-election-provider-staking-test"
```

```suggestion
name = "pallet-election-provider-e2e-test"
```

I think this is more accurate, these are testing this pallet, plus `staking` and a few more.